src/stpipe/cli/strun.py

```diff
@@ -21,7 +21,7 @@ def main():
         sys.exit(0)
 
     try:
-        Step.from_cmdline(sys.argv[1:])
+        cmdline.step_from_cmdline(sys.argv[1:])
```

Comment:
This change is not required by this PR (I'm fine removing it) but during debugging it was confusing that this interface calls `Step.from_cmdline` which then just calls `cmdline.step_from_cmdline`.
https://github.com/spacetelescope/stpipe/blob/799be658b5fbd36e1c4c47a07e00f76575b51d97/src/stpipe/step.py#L170-L191